moses/moses/main/populate_atomspace.cc

```diff
@@ -0,0 +1,62 @@
+#include <moses/data/table/table.h>
+#include <moses/data/table/table_io.h>
+#include <opencog/atomspace/AtomSpace.h>
+#include <moses/utils/value_key.h>
+#include <opencog/atoms/base/Link.h>
+#include "populate_atomspace.h"
+#include <opencog/atoms/value/Value.h>
+
+namespace opencog { namespace moses {
+
+void populate(AtomSpace *as, ITable &itable) {
+
+	std::vector<multi_type_seq>::const_iterator it, end;
+	int col_size = itable.get_types().size();
```

Comment:
My apologies if you took that name from my own code but I actually find `col_size` confusing because it suggests the size of a column. Maybe `cols_size` would be better cause it suggests we're talking about multiple columns, thus likely the number of columns. Alternatively different names like `n_columns` or `n_inputs` would be OK too, though I think `cols_size` is acceptable.

Reply:
I am pretty terrible at naming variables (or anything). so should it stay as it is or change? n_column is sounds very descriptive to me

Reply:
I have went with n_columns is that okay?

Reply:
`n_columns` is OK.